ee/packages/presence/src/Presence.ts

```diff
@@ -232,14 +232,19 @@ export class Presence extends ServiceClass implements IPresence {
 	}
 
 	private async validateAvailability(): Promise<void> {
-		if (this.hasLicense) {
+		if (this.hasLicense !== false) {
 			return;
 		}
 
 		if (this.getTotalConnections() > MAX_CONNECTIONS) {
 			this.broadcastEnabled = false;
 
 			await Settings.updateValueById('Presence_broadcast_disabled', true);
+		} else {
+			const presenceBroadcastDisabled = await Settings.findOneById('Troubleshoot_Disable_Presence_Broadcast');
```

Comment:
this looks expensive, doesn't it? potentially two trips for every new connection?

Reply:
Yeah, it did :grimacing: 
I've just had a better idea, I'll check the `broadcastEnabled` property to make sure that presence broadcast is always enabled when there's a valid license (so we just do it once, not at every new connection).